### DISCUSSION TITLE
I have error ERROR EACCES: permission denied, open '/root/.nuxtrc'

### DISCUSSION THREAD

**Discussion by ramcona** [nuxt/nuxt#23054] [Questions]
Hello i have will build some project and have error 

ERROR EACCES: permission denied, open '/root/.nuxtrc'

then i run a command "ls -alh /root/.nuxtrc" to check a path

ls: cannot access '/root/.nuxtrc': No such file or directory

have solution to fix it ?

**Comment by xjccc** [comment]
![image](https://github.com/nuxt/nuxt/assets/16478776/9eb85838-b39d-4122-95a8-3b2d40289afa)

ai's suggestion

**Reply by ramcona** [reply]
Hello, the problem is .nuxtrc not already

sudo chmod 644 /root/.nuxtrc
chmod: cannot access '/root/.nuxtrc': No such file or directory

sudo chown $USER:$USER /root/.nuxtrc
chown: cannot access '/root/.nuxtrc': No such file or directory

**Reply by xjccc** [reply]
nuxt2 or nuxt3?

**Comment by ramcona** [comment]
[SOLVE] with create file .nuxtrc on directory project..thanks god
